pkg/image/reference/reference.go

```diff
@@ -0,0 +1,51 @@
+package reference
+
+import (
+	"strings"
+
+	"github.com/docker/distribution/reference"
+)
+
+// NamedDockerImageReference points to a Docker image.
+type NamedDockerImageReference struct {
```

Comment:
This type is identical in everything except name to `DockerImageReference` in `pkg/image/api/types.go`. And `ParseNamedDockerImageReference` below is effectively wrapped by `pkg/image/api/helpers.go#ParseDockerImageReference`. Can we collapse the two types and functions into just 1 type and 1 function?